eoscostarica/lifebank
# Feat(frontend): add icons in offers

Resolves #605 

### What does this PR do?
A sponsor can choose among different icons to add to their offers

### How should this be tested?
- Make run
- Login as Sponsor 
- Add offer in Offers Management
- Check offers list

![image](https://user-images.githubusercontent.com/31549144/116482626-c026ac80-a842-11eb-987a-c403c457ebaa.png)


![image](https://user-images.githubusercontent.com/31549144/116482454-6b833180-a842-11eb-8e3d-dda9ce024781.png)
